README.md

```diff
@@ -22,13 +22,16 @@ real-time.
 
 ### Option 1: Self-host locally
 
-You can self-host Touca by running our install script that uses Docker Compose
-under the hood.
+You can self-host Touca using Docker Compose and through our CLI.
 
 ```bash
-/bin/bash -c "$(curl -fsSL https://touca.io/install.sh)"
+pip install touca
+touca server install
 ```
 
+Check out [our docs](https://touca.io/docs/server/self-hosting/) for other
+install options.
+
```

Comment:
Switching away from `install.sh` so we can remove it altogether in the future. Would love to hear if you agree with this proposal.